spec/services/decisions/case_decision_tree_spec.rb

```diff
@@ -139,6 +139,20 @@
 
       it { is_expected.to have_destination(:charges_summary, :edit, id: crime_application) }
     end
+
+    context 'and it is a change in financial circumstances application' do
+      let(:cifc?) { true }
+
+      before do
+        allow(FeatureFlags).to receive(:cifc_journey) {
+          instance_double(FeatureFlags::EnabledFeature, enabled?: true)
+        }
+
+        allow(subject).to receive(:requires_means_assessment?).and_return(false)
```

Comment:
Do we need to add a spec to confirm it routes to the employment status question if the app does require a means assessment